### PR TITLE
Update totals-footer-item.md

### PR DESCRIPTION
**Pull request by brentmackinnon**
replacing `<` & `>` with `&lt;` & `&gt;` for the purposes of displaying html within code snippets on woocommerce docs site.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.
2.
3.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: docs/cart-and-checkout-blocks/available-slot-fills.md
Comment (JacklynBiggin) [inline docs/cart-and-checkout-blocks/available-slot-fills.md:1]: The change on line 6 here shouldn't be there - this comment is needed to disable a linting rule.

File: docs/extension-development/settings-api.md
Comment (JacklynBiggin) [inline docs/extension-development/settings-api.md:1]: The <?php tags here (lines 72, 73, 75, 78, 77) need replacing too, to be safe.

**Review by JacklynBiggin — CHANGES_REQUESTED** [review]

File: docs/extension-development/settings-api.md
Comment (piinthecloud) [reply docs/extension-development/settings-api.md:1]: php tags work as expected. I loaded the gist with the manifest here so you can see the output: https://woo-docs-multi-develop.go-vip.net/docs/settings-api/

File: docs/cart-and-checkout-blocks/available-slot-fills.md
Comment (piinthecloud) [reply docs/cart-and-checkout-blocks/available-slot-fills.md:1]: updated ✅

**Review by JacklynBiggin — APPROVED** [review]

**Pull request merged**